### ISSUE TITLE
Adding a new module to address forms and spacing?

### ISSUE THREAD

**Issue by dominikwilkowski** [WestpacCXTeam/GUI-source#117]

**Comment by justinspencer** [comment]
Here's the markup of the example form showing the spacing in xs and md.
(sm and lg use the same spacing as md)

![forms-spacing-md](https://cloud.githubusercontent.com/assets/936597/11890006/cf604e38-a5a3-11e5-9b29-d0ec8b92c1a3.png)
![forms-spacing-xs](https://cloud.githubusercontent.com/assets/936597/11890007/cf634048-a5a3-11e5-99ae-1b08c462e086.png)

**Comment by justinspencer** [comment]
Is this issue closed?

**Comment by dominikwilkowski** [comment]
No. It was referenced by an issue that is now closed(WestpacCXTeam/GUI-docs#49). We needed a better idea on what we want in the module. Classes for spacing already exist so is it a form layout? If so what will it entail?

**Comment by justinspencer** [comment]
Here's the design and markup for the guts of a form. It contains 3 common patterns: Personal details, Contact details and Current address. We can extract whatever we need from this design to put into the GUI docs and the forms module.

[02-Form-Guts.pdf](https://github.com/WestpacCXTeam/GUI-source/files/292686/02-Form-Guts.pdf)
